src/Resources/app/storefront/dist/storefront/js/fin-search.js

```diff
@@ -1 +1 @@
erCategorySelect",d,"[data-filter-category-select]")},bEhy:function(e,t,n){"use strict";n.d(t,"a",(function(){return y}));var o=n("FGIj"),r=n("gHbT");function i(e){return(i="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(e){return typeof e}:function(e){return e&&"function"==typeof Symbol&&e.constructor===Symbol&&e!==Symbol.prototype?"symbol":typeof e})(e)}function c(e,t){for(var n=0;n<t.length;n++){var o=t[n];o.enumerable=o.enumerable||!1,o.configurable=!0,"value"in o&&(o.writable=!0),Object.defineProperty(e,o.key,o)}}function l(e,t){return!t||"object"!==i(t)&&"function"!=typeof t?function(e){if(void 0===e)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return e}(e):t}function u(e,t,n){return(u="undefined"!=typeof Reflect&&Reflect.get?Reflect.get:function(e,t,n){var o=function(e,t){for(;!Object.prototype.hasOwnProperty.call(e,t)&&null!==(e=a(e)););return e}(e,t);if(o){var r=Object.getOwnPropertyDescriptor(o,t);return r.get?r.get.call(n):r.value}})(e,t,n||e)}function a(e){return(a=Object.setPrototypeOf?Object.getPrototypeOf:function(e){return e.__proto__||Object.getPrototypeOf(e)})(e)}function s(e,t){return(s=Object.setPrototypeOf||function(e,t){return e.__proto__=t,e})(e,t)}var f,p,h,y=function(e){function t(){return function(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}(this,t),l(this,a(t).apply(this,arguments))}var n,o,i;return function(e,t){if("function"!=typeof t&&null!==t)throw new TypeError("Super expression must either be null or a function");e.prototype=Object.create(t&&t.prototype,{constructor:{value:e,writable:!0,configurable:!0}}),t&&s(e,t)}(t,e),n=t,(o=[{key:"_init",value:function(){u(a(t.prototype),"_init",this).call(this),this._validateMethods();var e=r.a.querySelector(document,this.options.parentFilterPanelSelector);this.listing=window.PluginManager.getPluginInstanceFromElement(e,"Listing"),this.listing.registerFilter(this),this._preventDropdownClose()}},{key:"_preventDropdownClose",value:function(){var e=r.a.querySelector(this.el,this.options.dropdownSelector,!1);e&&e.addEventListener("click",(function(e){e.stopPropagation()}))}},{key:"_validateMethods",value:function(){if("function"!=typeof this.getValues)throw new Error("[".concat(this._pluginName,'] Needs the method "getValues"\''));if("function"!=typeof this.getLabels)throw new Error("[".concat(this._pluginName,'] Needs the method "getLabels"\''));if("function"!=typeof this.reset)throw new Error("[".concat(this._pluginName,'] Needs the method "reset"\''));if("function"!=typeof this.resetAll)throw new Error("[".concat(this._pluginName,'] Needs the method "resetAll"\''))}}])&&c(n.prototype,o),i&&c(n,i),t}(o.a);h={parentFilterPanelSelector:".cms-element-product-listing-wrapper",dropdownSelector:".filter-panel-item-dropdown"},(p="options")in(f=y)?Object.defineProperty(f,p,{value:h,enumerable:!0,configurable:!0,writable:!0}):f[p]=h}},[["9TfS","runtime","vendor-node","vendor-shared"]]]);
```

Comment:
Why did this change?

Reply:
The old version was probably compiled with `6.3.3.0`, but locally now is `6.3.4.1`. Shopware requires you to re-compile your files when installing/uninstalling plugins in the development template. This is not required in production though.